remote-trait-object/src/forwarder.rs

```diff
@@ -18,13 +18,15 @@ use crate::packet::PacketView;
 use crate::port::Handler;
 use crate::service::Dispatch;
 use parking_lot::RwLock;
-use std::collections::HashMap;
+use std::collections::{HashMap, VecDeque};
 use std::fmt;
+use std::sync::Arc;
 
-type ServiceId = String;
+pub type ServiceObjectId = u32;
 
 pub struct ServiceForwarder {
-    service_handlers: RwLock<HashMap<ServiceId, Box<dyn Dispatch>>>,
+    service_handlers: RwLock<HashMap<ServiceObjectId, Arc<dyn Dispatch>>>,
```

Comment:
How about `service_objects`?
```suggestion
    service_objects: RwLock<HashMap<ServiceObjectId, Arc<dyn Dispatch>>>,
```